### ISSUE TITLE
Login page is invisible 

### ISSUE THREAD

**Issue by supersoulman** [kennethjiang/OctoPrint-Anywhere#55]
The login page for getanywhere.io on iOS safari is invisible. 

https://i.imgur.com/tQj7tLZ.jpg

**Comment by kennethjiang** [comment]
@supersoulman Sorry for the delay. Will take a look at it!

**Comment by kennethjiang** [comment]
@supersoulman I tried it on Safari on an iPhone 8 but was able to see the login page. What iPhone are you using?

**Comment by supersoulman** [comment]
I am using an iPhone 10. 
Here is a video of the issue. https://streamable.com/ijhl1

**Comment by supersoulman** [comment]
Okay just discovered, while typing this comment, if I just change tabs in safari and come back the login boxes appears. 

https://streamable.com/irctq

**Comment by kennethjiang** [comment]
Thank you so much @supersoulman for the gif. It made it clear to me what happened. 

I found a iphone x to try it and didn't have the same issue. So I'm not entirely sure why it's different on your iphone.

One thing that might help is: if you directly go to https://www.getanywhere.io/app/devices (I noticed that you went to https://www.getanywhere.io/pub first), you will directly go to the dashboard until your login session expires in 30 days. Hopefully this will make it less painful for you.
